content/millennium/r4/entities/organization.md

```diff
@@ -44,6 +44,14 @@ Search for Organizations that meet supplied query parameters:
  [`_count`]  | No                                      | [`number`] | The maximum number of results to return. Defaults to `100`.
  `name`      | This or `_id` or `identifier` or `type` | [`string`] | Name used for the organization.
 
+ Notes:
+
+- The `identifier` parameter
+  - May not be provided with any system information, only code details are provided. Search is performed across system's 
```

Comment:
Please update to: "Code details are required. System is optional. If system is not provided, search is performed across all systems supported by the Organization resource."

Thank you so much for drafting this up!

Reply:
Have updated the changes